lib/googlemaps.js

```diff
@@ -16,7 +16,8 @@ var _config = {
   'stagger-time': 200,
   'encode-polylines': true,
   'proxy': null,
-  'google-private-key': null
+  'google-private-key': null,
+  'key': null
```

Comment:
Since there's already a `google-private-key`, can this be renamed to add a little bit of distinction about what type of "key" it is? I see that the query param is just called `key`, but at least the config variable could be more specific, which would into make the if clauses a little bit more clear.